packages/liotad/README.md

```diff
@@ -1,43 +1,56 @@
-#Liota Package Manager
-Liota package manager consists of two parts: (1) A PackageThread that actually loads files, maintain global data structures and run package initialization/clean-up codes and (2) A PackageMessengerThread which listens on one or more specific communication channels, which is a named pipe for now, to provide with an interface for users and automated agents to send commands to package manager.
+# Liota Package Manager
+Liota package manager consists of two parts:
+
+(1) A PackageThread that actually loads files, maintain global data structures and run package initialization/clean-up logic and
+
+(2) A PackageMessengerThread which listens on one or more specific communication channels, which is a named pipe for now, to provide with an interface for users and automated agents to send commands to package manager.
 
 Package manager will initialize the data structures and threads when its own module (`package_manager.py`) is imported. Once imported, PackageMessengerThread will start listening on the pipe and PackageThread will load all packages specified in the auto list in a batch.
 
-##PackageThread
+## PackageThread
 
 Currently, supported commands include package action commands and statistical commands.
 
-###Package action commands
+### Package action commands
+
+* **load** package_name <sha1_checksum> package_name_2 <sha1_checksum_package_2>
```

Comment:
As "package_name_2 <sha1_checksum_package_2>" are not mandatory arguments, it will be good to write them in square brackets and mention they are optional. Here it feels that all these arguments are mandatory.

Reply:
I have mentioned below "even multiple packages can be loaded in the single command as shown above". It clarifies that these arguments are not mandatory.

I think the multiple package load should only be mentioned in comments putting [optional], "..."will confuse users that it is part of the command line.

Reply:
Ok, understood. Generally, the optional arguments are represented in square brackets. For example, in https://docs.docker.com/engine/reference/commandline/exec/
docker exec [OPTIONS] CONTAINER COMMAND [ARG...]
Similarly in https://kubernetes.io/docs/reference/kubectl/overview/
If you still think it is confusing, we can skip this.

Reply:
Personally, i don't think it is confusing. It definitely is not a standard, but a convention which is generally followed. Even linux man pages follow the same convention, so developers can easily understand it. We can address this later.